### DISCUSSION TITLE
About colors

### DISCUSSION THREAD

**Discussion by neuromagus** [jarun/nnn#1797] [Q&A] [answered]
About adding a custom file attributes for coloring.
NNN understand only standard file attributes from coreutils. How to add, for example, color for 'music', 'video', 'image' categories or maybe create a plugin for adding 'any' custom categories? Custom attributes? NNN has the ability to run suggested categories. It would be nice to add the ability to highlight colors.

**Comment by N-R-K** [comment, marked as answer]
You can compile with [icon support](https://github.com/jarun/nnn/wiki/Advanced-use-cases#file-icons) and customize the colors/icons in `src/icons.h`.
